Automattic/wp-calypso
# Correcting the feature list of the new plans page.

Related to Automattic/martech#1535

## Proposed Changes

This PR updates the feature list of the new plans grid according to Automattic/martech#1535. Citing below for easier reference:

* Unrestricted bandwidth currently applies to all our plans.
* ~The performance boosters lists applies to all our plans.~ The performance boosters applies to all our plans, except for the burst and CPUs as explained in pdgrnI-24M#comment-3414 
* Also true for “automatic datacenter fail-over”.
* Multi-site management is a feature of all our plans; same with “automated WordPress updates”.
* “DDOS mitigation” and “managed malware protection” are also present in all our plans, even if the malware one is less relevant on managed simple.
* Change “Advanced commerce solutions” to just “Commerce solutions”
* Let’s remove the “avant-garde design tools”.

Update on Feb. 21st: Per request of pdgrnI-24M#comment-3414, the upper plans table remains unchanged while we elaborate included features more in the plan comparison table below. 

## Testing Instructions



Access the new plans page at `/start/plans`, checking:

1. Whether the feature listed for each plan remains the same, except that "avant-garde design" has been removed. (updated per https://github.com/Automattic/wp-calypso/pull/73533/commits/d2511e138c62e8937bd1d17ebefef742c58f1125)
<img width="1291" alt="image" src="https://user-images.githubusercontent.com/1842898/220267320-0b51373a-9a62-4b71-8021-0f58f57cb8c4.png">

2. Whether the feature are correctly checked for each plan in the plans comparison table below.
<img width="1234" alt="image" src="https://user-images.githubusercontent.com/1842898/220053217-b5d56801-1a66-4b18-887d-ae79643eaaf6.png">
3. There should no longer be "avant-garde design tools"

4. "Advanced commerce solutions" should now be "Commerce solutions"

<img width="735" alt="image" src="https://user-images.githubusercontent.com/1842898/220053464-f09ecaaf-8fe6-4644-8d51-b33c11ca9170.png">


## Pre-merge Checklist
